src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,159 @@
+
+1. Point your web browser to the [CockroachDB {{ site.data.products.cloud }} Console](https://cockroachlabs.cloud) and [log in]({% link cockroachcloud/authentication.md  %}#console-ui-authentication).
+1. Select **Organization &raquo; Enterprise Licenses** from the nav bar dropdown. This will bring you to the **Enterprise Licenses** page, which shows a (possibly empty) list of licenses and their keys, with information about the status of each.
+1. Click the **Create License** button. This will bring you to a page called **Get started with CockroachDB Enterprise**.
+1. On this page, you will create an **Enterprise Trial** license or an **Enterprise Free** license.
+    1. To create an **Enterprise Trial** license:
+        1. Fill in the form with the required information.
+        1. Click the **Continue** button.
+        1. Select the checkbox to agree to the Terms & Conditions of the [CockroachDB Software License][csl].
+        1. Click the **Generate License Key** button. You will be redirected to the **Enterprise Licenses** page, where you can [start using the key](#set-a-license).
+    1. To create an **Enterprise Free** license:
+        1. Fill in the form with the required information.
+        1. Toggle the switch called **Find out if my company qualifies for an Enterprise Free license**. *By toggling this switch you are legally attesting to the fact that your company revenue meets the requirements of the license*.
+        1. Click the **Continue** button.
+        1. Select the checkbox to agree to the Terms & Conditions of the [CockroachDB Software License][csl].
+        1. Click the **Generate License Key** button. You will be redirected to the **Enterprise Licenses** page, where you can [start using the key](#set-a-license).
+
+{{site.data.alerts.callout_danger}}
+You will not be able to create more than one **Enterprise Trial** license per day. If you try, the UI will prevent you from proceeding, and the following message will be displayed:
+
+> A new Enterprise Trial license cannot be created at this time because one was recently created. Tell us more about your intended use to find out if you qualify for an Enterprise Free license, or create a new one after October 16, 2024 at 10:49 AM EDT.
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_success}}
+For quick local testing of Enterprise features, you can [run a single-node cluster](#single-node-clusters).
+{{site.data.alerts.end}}
+
+## Set a license
+
+{% include common/set-enterprise-license.md %}
+
+## Verify a license
+
+To verify a license, open the [built-in SQL shell]({% link {{ page.version.version }}/cockroach-sql.md %}) and use the [`SHOW CLUSTER SETTING`]({% link {{ page.version.version }}/show-cluster-setting.md %}) command to check the organization name and license key:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+>  SHOW CLUSTER SETTING cluster.organization;
```

Comment:
Does this need to be mentioned? I don't think it matters now because we no longer cross check the organization in the license with this setting.

Reply:
Ah sorry, holdover from before.  Removed in latest commit `Update with spilchen feedback (2)`  - thanks!